src/components/Navigator/QuickNavigationModal.vue

```diff
@@ -0,0 +1,236 @@
+<!--
+  This source file is part of the Swift.org open source project
+
+  Copyright (c) 2022 Apple Inc. and the Swift project authors
+  Licensed under Apache License v2.0 with Runtime Library Exception
+
+  See https://swift.org/LICENSE.txt for license information
+  See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+-->
+
+<template>
+  <div class="quick-navigation-modal">
+    <div class="container">
+      <div
+        class="close-icon"
+        @click="closeQuickNavigationModal()"
+      >
+        <p>
+          Close
+        </p>
+      </div>
+      <input
+        ref="input"
+        class="filter"
+        v-model="userInput"
+        type="text"
+      />
+      <div>
+        <div
+          v-for="(symbol, idx) in filteredSymbols"
+          class="symbol-match"
+          @click="closeQuickNavigationModal()"
+          :key="idx"
+        >
+          <Reference :url="symbol.path" :id="idx">
+              -
+              {{symbol.title.slice(0, symbol.start)}}
```

Comment:
```suggestion
              {{ symbol.title.slice(0, symbol.start) }}
```

Reply:
Fixed in [c2d8214](https://github.com/apple/swift-docc-render/pull/372/commits/c2d8214f63cd0570d57f672a4a6a75bc83f15f70)